### PR TITLE
chore(cas): Improve CAS logging

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

File: src/controllers/healthcheck-controller.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default class HealthcheckController {
 
       return res.status(OK).send()
     } catch (err) {
-      logger.err(err);
+      logger.err(`Performing healtcheck failed: ${err.message()}`);
```
Comment (valmack) [inline src/controllers/healthcheck-controller.ts:29]: `Healtcheck failed: ${err.message()}`

the "performing" is a bit confusing/misleading

File: src/services/anchor-service.ts [hunk 2]
```diff
@@ -305,10 +305,11 @@ export default class AnchorService {
 
     let request = null;
     for (let index = 0; index < requests.length; index++) {
+      let docId
       try {
         request = requests[index];
 
-        const docId = this._getRequestDocID(request)
+        docId = this._getRequestDocID(request)
```
Comment (valmack) [inline src/services/anchor-service.ts:312]: Why the change to let here?

File: src/services/anchor-service.ts [hunk 3]
```diff
@@ -317,7 +318,7 @@ export default class AnchorService {
         const candidate = new Candidate(new CID(request.cid), request.id, doc);
         groupedCandidates[candidate.docId] = groupedCandidates[candidate.docId] ? [...groupedCandidates[candidate.docId], candidate] : [candidate];
       } catch (e) {
-        logger.err(e);
+        logger.err(`Error while loading document ${docId.baseID.toString()} at commit ${docId.commit.toString}. Error: ${e.message}`)
```
Comment (valmack) [inline src/services/anchor-service.ts:321]: This message breaks if docId is null/undefined. We should probably throw in that case or have an additional try/catch for it above.

**Review by valmack — COMMENTED** [review]
Awesome! Some small comments

File: src/services/anchor-service.ts (hunk 2 above)
Comment (oed) [reply src/services/anchor-service.ts:312]: I guess because `docId` is used in the catch scope.

File: src/services/anchor-service.ts (hunk 3 above)
Comment (oed) [reply src/services/anchor-service.ts:321]: Just add some `?`

**Review by oed — APPROVED** [review]
lgtm, mod comments.

File: src/controllers/healthcheck-controller.ts (hunk 1 above)
Comment (valmack) [reply src/controllers/healthcheck-controller.ts]: Oh sorry @stbrody didn't see full context here. But your latest change makes sense.

File: src/controllers/anchor-controller.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { INTERNAL_SERVER_ERROR, OK } from 'http-status-codes';
 import { Request as ExpReq, Response as ExpRes } from 'express';
-import { Logger, Logger as logger } from '@overnightjs/logger';
+import { logger } from "../logger";
```
Comment (valmack) [inline src/controllers/anchor-controller.ts:3]: Small but would prefer to keep single vs double quotes consistent

**Review by valmack — APPROVED** [review]
Great!

File: src/controllers/anchor-controller.ts (hunk 4 above)
Comment (stbrody) [inline src/controllers/anchor-controller.ts:3]:
```suggestion
import { logger } from `../logger`;
```

Comment (stbrody) [reply src/controllers/anchor-controller.ts]: done
